godel_surface_detection/src/nodes/boundary_test_node.cpp

```diff
@@ -0,0 +1,72 @@
+#include <segmentation/surface_segmentation.h>
+#include <ros/ros.h>
+
+
+/*
+ * A stand-alone node that uses Godel's current surface segmentation algorithms
+ * for the purposes of testing and profiling
+ */
+
+
+static pcl::PointCloud<pcl::PointXYZRGB>::Ptr
```

Comment:
`SurfaceSegmentation` also does not use the typedef, will change later if you want.